packages/expo-firebase-polyfill/src/FirebasePolyfill.ts

```diff
@@ -0,0 +1,44 @@
+import { Platform } from 'react-native';
+
+import { btoa, atob } from './Base64';
+import { getRandomValues } from './Crypto';
+
+// Polyfill `btoa`
+// @ts-ignore
+if (!global.btoa) {
+  // @ts-ignore
+  global.btoa = btoa;
+}
+
+// Polyfill `atob`
+// @ts-ignore
+if (!global.atob) {
+  // @ts-ignore
+  global.atob = atob;
+}
+
+// Polyfill `Crypto.getRandomValues`
+// @ts-ignore
+if (!global.crypto || !global.crypto.getRandomValues) {
+  // @ts-ignore
+  global.crypto = global.crypto || {};
+  // @ts-ignore
+  global.crypto.getRandomValues = getRandomValues;
+}
```

Comment:
Seems like this should just be a crypto polyfill package. Maybe we could add "installPolyfill" to expo crypto and random instead.

Reply:
The crypto dependency has been removed from the Firebase JS SDK, so this code actually just got irrelevant. I'm remove that from the polyfill package.